templates/zerver/help/change-who-can-join-a-stream.md

```diff
@@ -1,75 +1,43 @@
 # Change the accessibility of a stream
 
 If you are a Zulip realm administrator, you can change who can join your stream
-by changing the stream's accessibility.
+by changing the stream's accessibility ***(By making it private or public)***.
+By making it private only the people you invite can join the stream. if it is
+public, anybody can join.
 
 Please note that any users that were previously subscribed to the stream will
 remain subscribed to the stream after you change the stream's accesibility.
 
-## Make a public stream private
-
 {!subscriptions.md!}
 
-3. Locate the public stream that you want to make private in the stream browser.
+3. Locate the stream that you want to change accessibilty in the stream browser.
```

Comment:
Um, I'm not sure about this, but is "want to change accessibility" proper English? I don't think "accessibility" can be used like this.